data_discovery_ai/pipeline.py

```diff
@@ -0,0 +1,89 @@
+import utils.preprocessor as preprocessor
+import model.keywordModel as model
+import utils.es_connector as connector
+import service.keywordClassifier as keywordClassifier
+import numpy as np
+import json
+
+
+def pipeline():
```

Comment:
all good but seem you missed my comment: https://github.com/aodn/backlog/issues/6006#issuecomment-2435001501

basic CI actions failed, so this can't be merged

second, please work on the latest `main`, your branch doesn't have the same base, because the changes are quite large, I'd recommend that you update your local `main` branch to be up-to-date, then create a new development branch, and move your changes in this PR to that new branch, and create a new PR, that will save you a lot more times than trying to `rebase`.

I will close this PR.

Reply:
no further changes required to this PR, please migrate/copy your changes in this PR to the new development branch that has up-to-date `main` base. @shaunahu

Reply:
ah looks like it does, sorry! please fix whatever that failed the CI action then